codex_of_the_damned/templates/archetypes/bibliodeque/midcap-dem.html

```diff
@@ -0,0 +1,12 @@
+{% extends "archetypes/weenie-dem.html" %}
+
+{% block archetype_name %}
+<h1><img src="{{ url_for('static', filename='img/icons/clan-malkavian-anti.svg') }}"> Midcap DEM</h1>
```

Comment:
Icon missing

Reply:
Icon to process : http://www.vekn.net/images/stories/icons/clans/iconclanmalkaviananti.gif